Jinhee/app.js

```diff
@@ -0,0 +1,93 @@
+require("dotenv").config();
+
+const express = require('express')
+const cors = require('cors')
+const logger = require('morgan'); 
+const app = express()
+
+app.use(cors())
+app.use(logger('dev'))
+app.use(express.json())
+
+const PORT = process.env.PORT
+
+const { DataSource } = require('typeorm');
+
+const appDataSource = new DataSource({
+    type: process.env.DB_CONNECTION,
+    host: process.env.DB_HOST,
+    port: process.env.DB_PORT,
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_DATABASE
+})
+
+
+
+appDataSource.initialize()
+    .then(() => {
+        console.log("Data Source has been initialized!") 
+    })
+    .catch((err) => {
+        console.error("Error during Data Source initialization", err)
+        appDataSource.destroy()
+    })
+
+
+
+app.get('/ping', function (req, res, next) {
+  res.json({message: 'rewe'}) 
+})
+
+app.post('/users/signup', async (req, res) => {
+	const { name, email, profileImage, password } = req.body
+    
+	await appDataSource.query(
+	`INSERT INTO users(
+	
+		name,
+    email,
+		profile_image,
+    password
+
+	) VALUES (?, ?, ?, ?);
+		`,
+		[name, email, profileImage, password]
+	); 
+     res.status(200).json({ message : "userCreated" });
+	})
```

Comment:
- github에 올라오는 코드가 어떠한 형태로 올라오는지 미리 꼼꼼히 확인하고 올리는 것은 개발자의 필수 소양입니다.
- CLI 환경에서 add 전, git diff 라는 명령어를 치면 실제 깃헙에서도 어떠한 양식으로 올라가는지 확인이 가능하니 참고해주세요!!

```suggestion
app.post('/users/signup', async (req, res) => {
	const { name, email, profileImage, password } = req.body
    
	await appDataSource.query(
	`INSERT INTO users(
	    name,
	    email,
	    profile_image,
	    password
	) VALUES (?, ?, ?, ?);
		`,
		[name, email, profileImage, password]
	); 
     res.status(200).json({ message : "userCreated" });
	})
```